### PR TITLE
RTC-671: Check position of callback method, first or latest

### PR DESCRIPTION
**Pull request by rbarros-daitan**

### CODE REVIEW

File: minuet/Paragon.Runtime/Plugins/PluginMethodV8Handler.cs [hunk 1]
```diff
@@ -63,11 +63,26 @@ protected override bool Execute(string name, CefV8Value obj, CefV8Value[] argume
...
+                    }
+                    else
+                    {
+                        if (arguments != null && arguments.Length != 0 && arguments[0].IsFunction)
+                        {
+                            v8Callback = arguments[0];
+                            parameters = v8Callback != null && arguments.Length > 1 ? arguments.Skip(1).ToArray() : arguments;
+                        }
+                        else
+                        {
```
Comment (lneir) [inline minuet/Paragon.Runtime/Plugins/PluginMethodV8Handler.cs:84]: is this else needed?

File: minuet/Paragon.Runtime/Plugins/PluginMethodV8Handler.cs [hunk 2]
```diff
@@ -63,11 +63,26 @@ protected override bool Execute(string name, CefV8Value obj, CefV8Value[] argume
...
+                    var parameters = arguments;
 
+                    if (arguments != null && arguments.Length != 0 && arguments[arguments.Length - 1].IsFunction)
+                    {
+                        v8Callback = arguments[arguments.Length - 1];
+                        parameters = v8Callback != null ? arguments.Take(arguments.Length - 1).ToArray() : arguments;
+                    }
+                    else
+                    {
+                        if (arguments != null && arguments.Length != 0 && arguments[0].IsFunction)
```
Comment (lneir) [inline minuet/Paragon.Runtime/Plugins/PluginMethodV8Handler.cs]: please add some documentation around what is happening here.

File: minuet/Paragon.Runtime/Plugins/PluginMethodV8Handler.cs (hunk 1 above)
Comment (rbarros-daitan) [reply minuet/Paragon.Runtime/Plugins/PluginMethodV8Handler.cs:84]: Yes, this is needed. This handler is used on all JS method calling. If we remove it, we may have problem with other methods.

**Comment by lneir** [comment]
lgtm

**Comment by arnaudbud** [comment]
thanks @lneir

**Pull request merged**